modules/ROOT/pages/rate-limiting-sla-policy.adoc

```diff
@@ -0,0 +1,218 @@
+
+You can also tweak the persistence frequency rate, which has a default of 10 seconds:
+
+`throttling.persistent_data_update_freq`
+
+[NOTE]
+
+Persistence is not available on CloudHub.
+
+
+== Configuring Policy Parameters
+
+When you apply the policy to your API from the UI, a list of parameters are displayed based on whether your environment includes Mule or non-Mule applications.
+
+=== Configuring Policy Parameters for Mule Clusters
+
+For Mule applications, the following parameters are displayed:
+
+[%header%autowidth.spread,cols="a,a,a"]
+|===
+|Parameter | Description | Example
+|Client ID Expression | A DataWeave expression that resolves the Client Application’s ID for a Contract of the API | Only one rate-limiting algorithm is created per ID:
+`#[attributes.headers['client_id']]`
+This example looks for an HTTP header called ‘client_id’ and uses its value.
+|Client Secret Expression | A DataWeave expression that resolves the Client application’s client secret for a contract of the API | 
+This is an optional value.
+Example: `#[attributes.headers['client_secret']]`
+|Clusterizable | The option that shares the quota defined for a group among all of the nodes of a Mule cluster| This feature requires the Mule runtime running as part of a Mule cluster.
+checked or unchecked
+| Expose Headers| Defines whether to expose the  <<x-ratelimit headers,response-header>> as part of the response | Checked or unchecked
+|===
+
+=== Configuring Policy Parameters for Anypoint Service Mesh (Non-Mule Applications)
+
+The Rate-Limiting SLA policy works in the same way for Anypoint Service Mesh (non-Mule applications), excluding the following differences:
+
+* The policy does not offer the option to expose headers
+* The client ID and client secret retrieving parameters differ, as explained in the following table.
+
+[%header%autowidth.spread,cols="a,a,a"]
+|===
+| Element | Description | Required
+| Credentials Origin 
+| Origin of the client ID and client secret credentials:
+
+* `client_id` and `client_secret` headers
+* Custom headers
+| Yes.
+
+| Client ID Header
+| The header name from where to extract the client ID in API requests
+| Yes, if you set the credentials origin to *Custom Headers*.
+
+| Client Secret Header
+| The header name from where to extract the client secret in API requests.
+| No.
+|===
+
+== FAQ
+
+*When does the window start?*
+
+The window starts with the first request after the policy is successfully applied.
+
+*What type of window does the algorithm use?*
+
+It uses a fixed window.
+
+*What happens when the quota is exhausted?*
+
+The algorithm is created on demand, when the first request is received. This event fixes the time window. Each request consumes the request quota from the current window until the time expires.
+
+When the request quota is exhausted, the Rate Limiting policy rejects the request. When the time window closes, the request quota is reset and a new window of the same fixed size starts.
```

Comment:
Shouldn't it be RL SLA here too? This one is incorrect in the main document too, sorry.

Reply:
Done! Thanks!